### DISCUSSION TITLE
Support of Angular v18 ?

### DISCUSSION THREAD

**Discussion by euskadi31** [vmware-clarity/ng-clarity#1422] [Q&A] [answered]
Clarity is compatible with Angular v18 ?

**Comment by repolevedavaj** [comment, marked as answer]
We did the upgrade and did not yet detect any issue.

I created a PR for this: https://github.com/vmware-clarity/ng-clarity/pull/1440

**Reply by kevinbuhmann** [reply]
I just merged this PR. It will be included in the next v17 release (v17.3.0).
